### DISCUSSION TITLE
Triagers

### DISCUSSION THREAD

**Discussion by PatheticMustan** [Prodigy-Hacking/ProdigyMathGameHacking#677] [(A) Anything]
It would be nice to have triagers to help manage issues and pull requests!
For now, we'll accept two people! If you would like to apply, comment or something

**Comment by ghost** [comment]
Can I apply?

**Reply by PatheticMustan** [reply]
Yes

**Reply by ghost** [reply]
Yay :D

**Reply by ghost** [reply]
Anything I need to do to apply?

**Comment by SakuraKitsunee** [comment]
@PatheticMustan  can i apply?

**Reply by SakuraKitsunee** [reply]
i am on this repository a lot so i would be good for the job!

**Comment by irhrhd** [comment]
I would also like to apply!

**Comment by ArcerionDev** [comment]
Just to clarify-
This will give you the ability to close or lock issues and pull requests, just to root out duplicates and invalids.

**Reply by SakuraKitsunee** [reply]
ok thats what i thought

**Comment by irhrhd** [comment]
Yes Sir I understand

**Comment by SakuraKitsunee** [comment]
yes, i understand too!

**Comment by SakuraKitsunee** [comment]
do we have to fill out a form or what...

**Reply by ArcerionDev** [reply]
just ask here, we'll accept you if we think you're fit

**Reply by SakuraKitsunee** [reply]
Ok i'm asking! i really want to help PMGH in anyway I can

**Reply by ArcerionDev** [reply]
@CRobbins0867 I know that @irhrhd doesn't have access to a device that is able to install the hacks, do you

**Reply by SakuraKitsunee** [reply]
yes i do

**Reply by ArcerionDev** [reply]
alright, accepting @CRobbins0867 as the second and last triager, I think it's best if our issue managers are able to duplicate issues and bugs and such

**Reply by SakuraKitsunee** [reply]
its my personal computer (im not on it rn tho)

**Reply by ArcerionDev** [reply]
alright, you should have an invite by email.

**Reply by SakuraKitsunee** [reply]
tysm!!

**Reply by SakuraKitsunee** [reply]
i accepted it!

**Reply by ArcerionDev** [reply]
make sure to manage new issues or clear out invalid ones starting now, then
